app/imports/ui/layouts/CompanyProtectedRoute.jsx

```diff
@@ -22,7 +22,7 @@ const CompanyProtectedRoute = ({ ready, children }) => {
     return <LoadingSpinner />;
   }
   const isAdmin = Roles.userIsInRole(Meteor.userId(), 'admin');
-  const isCompany = Roles.userIsInRole(Meteor.userId(), 'company');
+  const isCompany = Roles.userIsInRole(Meteor.userId, 'company');
```

Comment:
Change this back. The correct usage to invoke the method is `Meteor.userId()`. If there is an issue with the `CompanyProtectedRoute` component, this is not the solution.